ament_flake8/ament_flake8/main.py

```diff
@@ -199,7 +210,8 @@ def parse_config_file(config_file):
         opts_manager = manager.OptionManager(
             version=flake8_version,
             plugin_versions='',
-            parents=[]
+            parents=[],
+            formatter_names=[],
```

Comment:
I don't think that this works for version 5 of flake8: https://github.com/PyCQA/flake8/blob/5.0.4/src/flake8/options/manager.py#L318-L323

So I think we need another conditional here which adds this for version 6, but not for version 5.

Reply:
I tried building and testing rolling with this change with flake8 v5 and then separately v6 in my python environment and it seems to work. Is flake8 vendored in ros? Is there some way to know which flake8 it is using?

In the case of v6 and v5 I see this warning:

```
test/test_flake8.py::test_flake8
  /usr/lib/python3/dist-packages/flake8/plugins/manager.py:261: DeprecationWarning: SelectableGroups dict interface is deprecated. Use select.
    eps = importlib_metadata.entry_points().get(self.namespace, ())

-- Docs: https://docs.pytest.org/en/stable/warnings.html
---
```

Reply:
I just tested this out locally, and indeed this did not work with flake8 5 anymore.  I've now pushed fdb6977984986fb94e3e083095e997ee5c010770, which fixes that problem.